lib/pages/chat/chat.dart

```diff
@@ -895,7 +895,7 @@ class ChatController extends State<Chat>
       setState(() {});
       return;
     }
-    await scrollToIndex(eventIndex, highlight: highlight);
+    await scrollToIndex(eventIndex + 1, highlight: highlight);
```

Comment:
why `+1`? @sherlockvn @nqhhdev who can refactor it?

Reply:
@sherlockvn will refactor it, pls @drminh2807 remove it.

Reply:
okay, after planning